dvc/repo/checkout.py

```diff
@@ -21,7 +21,8 @@ def checkout(self, target=None, with_deps=False, force=False, recursive=False):
             stages = self.collect(target, with_deps=with_deps)
         except (StageFileDoesNotExistError, StageFileBadNameError) as exc:
             raise DvcException(
-                str(exc) + " Did you mean 'git checkout {}'?".format(target)
+                str(exc) + " Did you mean 'git checkout {}'?".format(target),
+                cause=exc,
```

Comment:
I think it'll make it repeat the error message (`str(exc)` already includes some information about the cause)

Reply:
Sounds strange for me, I'll check.

Reply:
No, it is ok. Or I didn't understand what confuses you.

Reply:
```(.env) [ivan@ivan ~/Projects/test-metrics]$ dvc checkout test
Error: unexpected error - 'test' does not exist. Did you mean 'git checkout test'?: 'test' does not exist.
```
'test' does not exist ... 'test' does not exist

Reply:
Oh. Hmm...

Reply:
Ok, lets postpone this to another pull-request if it would be reasonable. (made a separate commit just for this :) )